build.bat

```diff
@@ -36,16 +36,16 @@ exit /b
 cd /d %~dp0
 mkdir tools >NUL 2>&1
 echo Downloading rsrc...
-curl -sSfL https://github.com/akavel/rsrc/releases/download/v0.10.2/rsrc_windows_amd64.exe -o tools\rsrc.exe
-echo Compiling All Resources...
+curl -sSfLO https://github.com/akavel/rsrc/releases/download/v0.10.2/rsrc_windows_amd64.exe -o tools\rsrc.exe
```

Comment:
I don't think the "O" option can be used with the "o" option...

Reply:
Oops! My bad😭Sorry..I'll fix it right away....